GhidraJupyterKotlin/src/main/java/GhidraJupyterKotlin/KernelThread.java

```diff
@@ -0,0 +1,9 @@
+package GhidraJupyterKotlin;
+
+import ghidra.util.task.MonitoredRunnable;
+
+import java.io.File;
+
+public interface KernelThread extends MonitoredRunnable {
+    File getConnectionFile();
```

Comment:
I always wonder which is clearer - `ConnectionFile()` or `getConnectionFile()`...

Reply:
Kotlin automagically adds a fake attribute `.connectionFile` if there is a method `getConnectionFile()` so this is IMO nicer and more "standard"